packets/serverbound/chunkData.js

```diff
@@ -0,0 +1,23 @@
+const ByteStream = require('../../byteStream.js');
+const Packet = require('../../packet.js');
+
+class ChunkData extends Packet {
+  constructor(packet){
+    super()
+    Object.assign(this, packet)
+    var bs = new ByteStream(Buffer.from(this.dataBuffer))
+    this.x = bs.readInt()
+    this.z = bs.readInt()
+  }
+
+  localize(x, z) {
+    this.x = x - this.x
```

Comment:
Move the given chunk to the appropriate location given where we've placed it on our map.